apps/webapp/src/components/shared/select-token-modal.tsx

```diff
@@ -11,7 +11,7 @@ import {
 import { fromBaseUnitAmount } from '@penumbra-zone/types';
 import { cn } from '@penumbra-zone/ui/lib/utils';
 import { AccountBalance } from '../../fetchers/balances';
-import { AssetIcon } from './asset-icon';
+import { AssetIcon } from './asset-icon.tsx';
```

Comment:
Is this intentional?

I've noticed that, throughout the web monorepo, we're importing from `some-file.tsx` in some places, and just `some-file` in others. Any idea why that is? Feels like a mistake.